src/test/java/com/google/api/codegen/testdata/csharp_gapic_snippets_library.baseline

```diff
@@ -141,6 +171,38 @@ namespace Google.Example.Library.V1.Snippets
             // End snippet
         }
 
+        public async Task GetShelfAsync_RequestObject()
+        {
+            // Snippet: GetShelfAsync(GetShelfRequest,CallSettings)
+            // Create client
+            LibraryServiceClient libraryServiceClient = LibraryServiceClient.Create();
+            // Initialize request argument(s)
+            GetShelfRequest request = new GetShelfRequest
+            {
+                Name = new ShelfName("[SHELF_ID]").ToString(),
```

Comment:
I think it would be better to generate snippets using the added properties, e.g.

```csharp
ShelfName = new ShelfName(...)
```

but that can *definitely* come as a follow-up